### PR TITLE
Add ops-metric-client cli

### PR DESCRIPTION
**Pull request by zgalor**
Add 2 clis that use metric_sender class to send metrics:
ops-metric-client.py
ops-metric-gpg-client.py.

PR depends on https://github.com/openshift/openshift-tools/pull/1511

### CODE REVIEW

**Comment by zgalor** [comment]
@twiest PTAL

File: scripts/monitoring/ops-metric-client.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+        metric_verbose = self.args.verbose
+        metric_debug = self.args.debug
+        if isinstance(metric_verbose, str):
+            metric_verbose = (metric_verbose == 'True')
+
+        if isinstance(metric_debug, str):
+            metric_debug = (metric_debug == 'True')
+
+        self.metric_sender = MetricSender(self.args.config_file, host=host, verbose=metric_verbose, debug=metric_debug)
```
Comment (joelddiaz) [inline scripts/monitoring/ops-metric-client.py]: @zgalor config_file parameter is no longer the first parameter to MetricSender

Comment (zgalor) [reply scripts/monitoring/ops-metric-client.py]: fixed

**Comment by zgalor** [comment]
Closing this - content already in https://github.com/openshift/openshift-tools/pull/1650

**Pull request closed without merging**